scripts/windows/orange-conda.nsi

```diff
@@ -644,51 +658,51 @@ Section -Register SectionRegister
 
     ${LogWrite} "Register .ows filetype"
     WriteRegStr SHELL_CONTEXT \
-        "Software\Classes\.ows" "" "OrangeCanvas"
+        "Software\Classes\.ows" "" ${INSTALL_REGISTRY_KEY}
     WriteRegStr SHELL_CONTEXT \
-        "Software\Classes\OrangeCanvas" "" "Orange Workflow"
+        "Software\Classes\${INSTALL_REGISTRY_KEY}" "" "Orange Workflow"
     WriteRegStr SHELL_CONTEXT \
-        "Software\Classes\OrangeCanvas\DefaultIcon" "" \
-        "$PythonPrefix\share\orange3\icons\OrangeOWS.ico"
+        "Software\Classes\${INSTALL_REGISTRY_KEY}\DefaultIcon" "" \
+        "$PythonPrefix\share\${ICONDIR}\OrangeOWS.ico"
     WriteRegStr SHELL_CONTEXT \
-        "Software\Classes\OrangeCanvas\Shell\Open\Command\" "" \
-        '"$PythonExecPrefix\pythonw.exe" -m Orange.canvas "%1"'
+        "Software\Classes\${INSTALL_REGISTRY_KEY}\Shell\Open\Command\" "" \
+        '"$PythonExecPrefix\pythonw.exe" -m "{LAUNCHERMODULE} "%1"'
```

Comment:
This `-m "{LAUNCHERMODULE} "%1"` should be `-m ${LAUNCHERMODULE} "%1"`

Reply:
Thanks. I have unified this line to match others as well ("" around -m flag). Is that ok or should it be avoided in some cases?

Reply:
No. The quoting here should be avoided.